contrib/jenkins/run_e2e.sh

```diff
@@ -94,7 +92,9 @@ ${ROOT}/contrib/jenkins/install_catalog.sh ${ARGUMENTS} \
 make bin/e2e.test \
   || error_exit "Error when making e2e test binary."
 
-KUBECONFIG="${KUBECONFIG}" SERVICECATALOGCONFIG="${SC_KUBECONFIG}" ${ROOT}/bin/e2e.test \
+KUBECONFIG="${KUBECONFIG}" ${ROOT}/bin/e2e.test \
```

Comment:
Nope, it's not, the service catalog context stuff is split up onto the lines below because it got too long.